browse/services/document/metadata.py

```diff
@@ -490,6 +490,8 @@ def parse_abs_file(filename: str) -> DocMetadata:
         # there are two main components to an .abs file that contain data,
         # but the split must always return four components
         components = RE_ABS_COMPONENTS.split(raw)
+        if len(components) > 4:
+            components = alt_component_split(components)
```

Comment:
`alt_component_split()` is only used when the usual parse fails. This is to avoid introducing any bugs to the existing parse code.